### ISSUE TITLE
Add custom reducers

### ISSUE THREAD

**Issue by reminyborg** [reminyborg/aggsy#5]
Need to find a good way of adding custom reducers.

The alternative i am liking is having an options object that you add when you call aggsy.

``` javascript
var reducers: {
  '_last': function (prev, curr) { return curr }
}
aggsy(query, { reducers: reducers }, data)
```

The good thing about this is that we could use the options object in the future for other things.

**Comment by reminyborg** [comment]
Will implement with a small change in parameter order.

``` javascript
// Aggsy parameters in the order of most used:
aggsy(query, data)
// or
aggsy(query, data, options)
// or
aggsy(query, options)
```

**Comment by reminyborg** [comment]
Solved in v1.0.0
